spec/views/plantings/show.html.haml_spec.rb

```diff
@@ -39,12 +39,6 @@ def create_planting_for(member)
       )
     end
 
-    it "shows the sunniness" do
-      render
-      rendered.should have_content 'Sun or shade?'
-      rendered.should have_content 'sun'
-    end
-
```

Comment:
I can do that, but it says in the file that the test is deprecated. Will it be okay if add some feature tests instead?